.github/workflows/ci_cd.yml

```diff
@@ -236,7 +236,7 @@ jobs:
   embedding-tests:
     name: Embedding testing and coverage
     runs-on: ubuntu-latest
-    timeout-minutes: 10
+    timeout-minutes: 15
```

Comment:
Why are we changing this in this PR?

Reply:
I am just testing it to see what the failure is, currently other embedding tests are passing. some of them automatically failing because of the 10 mins limit.